### PR TITLE
Add --network option to installer

### PR DESCRIPTION
**Pull request by icetan**
## Merge Criteria

- [x] install `omnia` as a *feed* on a clean VPS using `install-omnia`
  - [x] make sure `systemctl status omnia` is `active` and hasn't exited
  - [x] check that output from `journalctl -e -u omnia` looks reasonable
  - [x] make sure `systemctl status ssb-server` is `active` and hasn't exited
  - [x] check that output from `journalctl -e -u ssb-server` looks reasonable
  - [x] reboot VPS and make sure services auto-start
- [x] install `omnia` as a *relayer* on a clean VPS using `install-omnia`
  - [x] make sure `systemctl status omnia` is `active` and hasn't exited
  - [x] check that output from `journalctl -e -u omnia` looks reasonable
  - [x] make sure `systemctl status ssb-server` is `active` and hasn't exited
  - [x] check that output from `journalctl -e -u ssb-server` looks reasonable
  - [x] reboot VPS and make sure services auto-start

### CODE REVIEW

File: systemd/install-omnia [hunk 1]
```diff
@@ -17,9 +17,10 @@ Commands:
...
 
 Options:
-  --from ADDRESS                  Set Ethereum address to use for signing
-  --keystore PATH                 Set Ethereum keystore path
-  --password PATH                 Set Ethereum keystore password file path
+  --from         ADDRESS          Set Ethereum address to use for signing
+  --keystore     PATH             Set Ethereum keystore path
+  --password     PATH             Set Ethereum keystore password file path
+  --network      ETH_NETWORK      Set relayer Ethereum network
   --ssb-caps     CAPS_FILE        Set Scuttlbot caps from file
```
Comment (NiklasKunkel) [inline systemd/install-omnia:24]:
```suggestion
  --ssb-caps     CAPS_FILE        Set Scuttlebot caps from file
```

File: systemd/install-omnia [hunk 2]
```diff
@@ -17,9 +17,10 @@ Commands:
...
   --ssb-external EXTERNAL_ADDRS   Set Scuttlbot external IP/host address
```
Comment (NiklasKunkel) [inline systemd/install-omnia:25]:
```suggestion
  --ssb-external EXTERNAL_ADDRS   Set Scuttlebot external IP/host address
```

File: systemd/install-omnia [hunk 3]
```diff
@@ -17,9 +17,10 @@ Commands:
...
   --ssb-secret   SECRET_FILE      Install Scuttlbot secret
```
Comment (NiklasKunkel) [inline systemd/install-omnia:26]:
```suggestion
  --ssb-secret   SECRET_FILE      Install Scuttlebot secret
```

**Comment by icetan** [comment]
> very minor typos but apart form that all good.

Thanks, I always make that typo and never see it :smile:

**Pull request merged**